app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -1307,7 +1307,7 @@ class Settings(private val appContext: Context) : PreferencesHolder {
      */
     var enableTaskContinuityEnhancements by featureFlagPreference(
         key = appContext.getPreferenceKey(R.string.pref_key_enable_task_continuity),
-        default = false,
+        default = true,
```

Comment:
Maybe we should just set this to `FeatureFlags.taskContinuityFeature`. I saw this done elsewhere.